### PR TITLE
ISPN-9442 Scripts with JSON datatype should be able to read data as S…

### PR DESCRIPTION
**Pull request by gustavonalle**
…tring by default

https://issues.jboss.org/browse/ISPN-9442

### CODE REVIEW

File: client/hotrod-client/src/test/resources/json-script.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+// mode=local,language=javascript,datatype='application/json; charset=utf-8'
+var valueJSON = {"v":"value"}
+cache.put("key", JSON.stringify(valueJSON));
+
+var value = JSON.parse(cache.get("key"))
+value.v = "value2"
+
+cache.put("key2", JSON.stringify(value))
+cache.get("key2")
```
[the remark this answers is not in the document]
Comment (gustavonalle) [reply client/hotrod-client/src/test/resources/json-script.js]: fixed

**Comment by anistor** [comment]
LGTM but CI build died unexpectedly. I'll re-trigger.

**Comment by gustavonalle** [comment]
Rebased, waiting for CI...

**Comment by gustavonalle** [comment]
@anistor CI approved

**Comment by anistor** [comment]
Integrated in master. Thanks @gustavonalle !

**Pull request closed without merging**